gerousiabot/bot_handlers.py

```diff
@@ -55,6 +75,15 @@ def get_user_list() -> list:
     return user_list
 
 
```

Comment:
Sees check_user function .... does not see check_user function being used.

Sees duplication of code in check_user function below if query.data == Response.CHECK_USERS.name .

Maybe i need sleep but this looks off.

Reply:
Original intent was to call that function in response to the keyboard button or a direct /checkusers command. After I implemented the button logic I had forgotten to remove the redundancy 🤦🏾‍♂️